### PR TITLE
Handle interface name to table name

### PR DESCRIPTION
**Pull request by matt-mazzucato**
Given the possibility to include hyphens into interface_name, handle the mapping of interface_name to table name.

### CODE REVIEW

**Comment by matt-mazzucato** [comment]
Address #96

**Comment by zanettea** [comment]
If hyphens are simply replaced with spaces, then the poor user will end up striving to understand why the service refuses to install two interfaces with different names saying the interface already exists.

Either you should improve the error or (better imho) provide a transparent mapping which does not fail in corner cases.

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [0.11.0-beta.2] - Unreleased
+### Changed
+- [realm_management] Handle hyphens in `interface_name`. ([96](https://github.com/astarte-platform/astarte/issues/96))
```
Comment (rbino) [inline CHANGELOG.md]: Missing # before the issue number

**Comment by matt-mazzucato** [comment]
@zanettea Thanks for the input. I improved the error message.
Unfortunately it's not possible to employ a bijective function while mapping interface_name to table name: this is due to Cassandra's constraints as discussed in [this issue](https://github.com/astarte-platform/astarte_core/issues/7#issuecomment-567896895)

**Pull request merged**